doc/source/metadata/temp12max_perc_ncdump.txt

```diff
@@ -0,0 +1,82 @@
+netcdf PERC-20221222T1800Z-B20221220T0615Z-temperature_at_screen_level_max-PT12H {
```

Comment:
I think your file has a `PERC-` prefix which shows up weirdly here. Not a major issue.

Reply:
I've removed these prefixes from the sample code now.